user-service/src/main/java/rs/edu/raf/banka1/services/implementations/OrderServiceImpl.java

```diff
@@ -68,7 +69,7 @@ public void startOrder(final Long orderId) {
             return;
 
         final ListingBaseDto listingBaseDto = marketService.getStock(marketOrder.getStockId());
-        Random random = new Random();
```

Comment:
kad resis neki komentar treba da markiras resolved da bimo znali da je to reseno